packages/api-client/src/components/AddressBar/AddressBarServer.vue

```diff
@@ -57,16 +57,26 @@ const handleAddServer = () =>
   commandPaletteBus.emit({
     commandName: 'Add Server',
   })
+
+const serverUrlWithoutTrailingSlash = computed(() => {
+  if (activeServer.value?.url?.endsWith('/')) {
+    return activeServer.value.url.slice(0, -1)
+  }
+  return activeServer.value?.url || ''
+})
```

Comment:
nit: slightly cleaner if u like

```suggestion
const serverUrlWithoutTrailingSlash = computed(() => activeServer.value?.url?.replace(/\/^/, '') || '')
```